core/players-handbook-2024/races/race-goliath.xml

```diff
@@ -0,0 +1,149 @@
+			<ul class="unstyled">
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Cloud’s Jaunt (Cloud Giant).</strong> As a Bonus Action, you magically teleport up to 30 feet to an unoccu­pied space you can see.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Fire’s Burn (Fire Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Frost’s Chill (Frost Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal ld6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Hill’s Tumble (Hill Giant).</strong> When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Stone’s Endurance (Stone Giant).</strong> When you take damage, you can take a Reaction to roll ldl2. Add your Constitution modifier to the number rolled and reduce the damage by that total.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Storm’s Thunder (Storm Giant).</strong> When you take damage from a creature within 60 feet of you, you can take a Reaction to deal ld8 Thunder dam­age to that creature.</li>
+			</ul>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Racial Trait" name="Giant Ancestry" supports="Giant Ancestry 2024" />
+		</rules>
+	</element>
+	<element name="Cloud’s Jaunt (Cloud Giant)" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY_CLOUDS_JAUNT">
+		<supports>Giant Ancestry 2024</supports>
+		<description>
+			<p>As a Bonus Action, you magically teleport up to 30 feet to an unoccu­pied space you can see.</p> 
+		</description>
+		<sheet alt="Cloud’s Jaunt" action="Bonus Action" usage="{{proficiency}}/Long Rest">
+			<description>You magically teleport up to 30 feet to an unoccu­pied space you can see.</description>
+		</sheet>
+	</element>
+	<element name="Fire’s Burn (Fire Giant)" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY_FIRES_BURN">
+		<supports>Giant Ancestry 2024</supports>
+		<description>
+			<p>When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</p> 
+		</description>
+		<sheet alt="Fire’s Burn" usage="{{proficiency}}/Long Rest">
+			<description>When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</description>
+		</sheet>
```

Comment:
seems like anywhere 1d# was copied from your source, it translated it into letters instead of numbers

```suggestion
		<description>
			<p>When you hit a target with an attack roll and deal damage to it, you can also deal 1d10 Fire damage to that target.</p> 
		</description>
		<sheet alt="Fire’s Burn" usage="{{proficiency}}/Long Rest">
			<description>When you hit a target with an attack roll and deal damage to it, you can also deal 1d10 Fire damage to that target.</description>
		</sheet>
```